psutil/arch/windows/security.c

```diff
@@ -160,19 +159,17 @@ psutil_set_privilege(HANDLE hToken, LPCTSTR Privilege, BOOL bEnablePrivilege) {
 int
 psutil_set_se_debug() {
     HANDLE hToken;
-    if (! OpenThreadToken(GetCurrentThread(),
+    if (!OpenProcessToken(GetCurrentProcess(),
                           TOKEN_ADJUST_PRIVILEGES | TOKEN_QUERY,
-                          FALSE,
```

Comment:
mmm this looks wrong; aren't you now passing 3 args instead of 4?

Reply:
I'm using GetProcessToken instead of GetThreadToken and it needs 1 less argument
In the case where the actual privileged call is performed from another thread than the one getting th SE_DEBUG privilege, it might be best to have the whole process enable the privilege

Reply:
I see (didn't notice you changed syscall). I checked ProcessHacker source code and it also uses `OpenProcessToken` so I guess it makes sense to do the same. Good one.